lib/handlers/get.js

```diff
@@ -109,6 +115,11 @@ function get(req, res, includeBody) {
             }
         }
 
+        // redirect to file browser if we got text/html with highest priority
+        if (res.acceptTypes.indexOf('text/html') === 0) {
+            return res.redirect(303, browseSkin + req.protocol + '/' + req.get('host') + req.path);
```

Comment:
This won't work when ldnode is mounted on a route.
this should be instead:

```
res.redirect(303, browseSkin + req.protocol + '/' + req.get('host') + req.originalUrl);
```

`req.path` to `req.originalUrl`